code/game/machinery/computer/medical.dm

```diff
@@ -272,12 +272,14 @@
 							if(!canUseMedicalRecordsConsole(usr, t1, a1))
 								return
 							src.active1.fields["fingerprint"] = t1
-					if("sex")
+					if("gender")
```

Comment:
Can we make this a drop down box instead, is more convenient if there are more than 2 options

Reply:
Done, made it the same for the security records console.